tests/test_search_adaptors.py

```diff
@@ -60,7 +60,7 @@ def test_vespa_search_adaptor__works(fake_vespa_credentials):
 )
 @pytest.mark.vespa
 def test_vespa_search_adaptor__is_fast_enough(fake_vespa_credentials, params):
-    MAX_SPEED_MS = 750
+    MAX_SPEED_MS = 850
```

Comment:
why has this change been made as part of this PR?

Reply:
Hello! 

Are you recommending we make a fixture for the variable or something? 

It's effectively just a value only used here that helps us understand if we introduce a change that greatly affects response time. The issue is - response time is very variable in CI for some reason so it's not actually that useful as can be seen in the screen shots, upping it here seems to make the tests fail ~20% of the time rather than ~50%

Reply:
oh cool, i just wanted to check whether it was needed here but that CI drive-by improvement makes sense